bucket/potplayer.json

```diff
@@ -0,0 +1,71 @@
+{
+    "version": "1.7.20538",
+    "description": "Highly customizable media player",
+    "homepage": "https://potplayer.daum.net/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://potplayer.daum.net/publicRelation"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://t1.daumcdn.net/potplayer/PotPlayer/Version/20190919_1.7.20538/PotPlayerSetup64.exe#/cosi.7z",
+            "hash": "c54f447ccd217cb07aca6199722054dd0e2ae97b07aa946192cd4e21b3db7b68",
+            "shortcuts": [
+                [
+                    "PotPlayer64.exe",
+                    "PotPlayer"
+                ],
+                [
+                    "PotPlayerMini64.exe",
+                    "PotPlayer Mini"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://t1.daumcdn.net/potplayer/PotPlayer/Version/20190919_1.7.20538/PotPlayerSetup.exe#/cosi.7z",
+            "hash": "c7e7ee44e45cb7d09eac7a25c7710cb619eb6759df6a05439aa2d584233b9770",
+            "shortcuts": [
+                [
+                    "PotPlayer.exe",
+                    "PotPlayer"
+                ],
+                [
+                    "PotPlayerMini.exe",
+                    "PotPlayer Mini"
+                ]
+            ]
+        }
+    },
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\`$*\" -Force -Recurse"
+        ]
+    },
+    "persist": [
+        "Capture",
+        "Extention",
+        "IconPack",
+        "Logos",
+        "Playlist",
+        "Skins",
+        "UrlList",
```

Comment:
File or folder?

Reply:
All folders except for .ini files

Reply:
Do not mark conversation as resolved when second side of conversation did not read it.

When you mark it and newer notification is sent it is hidden and you have to go through all collapsed conversations.

Reply:
Sorry, didn't know that